src/Pages/Templates/TemplatesTable/components/AddTemplate/steps/SetUpDateStep.tsx

```diff
@@ -128,122 +123,58 @@ export default function SetUpDateStep() {
           id='use snapshot date radio'
           ouiaId='use-snapshot-date-radio'
           name='use-snapshot-date'
-          label='Use a snapshot'
-          description='Use content from snapshots up to a specific date.'
+          label='Use up to specific date'
+          description='Includes repository changes up to this date.'
           isChecked={!templateRequest.use_latest}
           onChange={() => {
             if (templateRequest.use_latest) {
               setTemplateRequest((prev) => ({ ...prev, use_latest: false, date: '' }));
             }
           }}
         />
+        <DatePicker
+          value={templateRequest.date ?? ''}
+          required
+          requiredDateOptions={{ isRequired: true }}
```

Comment:
i notice if i select `Use up to specific date` and click inside the date picker and enter an invalid date (or leave it empty) then instead decide to select `Use latest content`, the `Date cannot be blank` error remains but I can proceed in the wizard. i'd expect the error about the invalid date to disappear if `Use latest content` is selected. 

i wonder if this is because a date is set to be required here?